src/native/libs/System.IO.Compression.Native/pal_zlib.c

```diff
@@ -44,6 +39,10 @@ Initializes the PAL_ZStream by creating and setting its underlying z_stream.
 static int32_t Init(PAL_ZStream* stream)
 {
     z_stream* zStream = (z_stream*)calloc(1, sizeof(z_stream));
+
+	zStream->zalloc = z_custom_calloc;
+	zStream->zfree = z_custom_cfree;
```

Comment:
use spaces instead of tabs to match the rest of the file
```suggestion
    zStream->zalloc = z_custom_calloc;
    zStream->zfree = z_custom_cfree;
```

Reply:
Agh yes, I don't know how those tabs got added. My VS Code is trolling me.